spec/models/web_search_spec.rb

```diff
@@ -67,149 +65,133 @@
                               per_page: 10,
                               query: 'government (site:gov OR site:mil)'))
 
-        described_class.new @valid_options
+        described_class.new valid_options
       end
     end
-
   end
 
   describe '#cache_key' do
-    before do
-      @valid_options = {query: 'government', affiliate: affiliate, page: 5}
+    let(:valid_options) do
+      { query: 'government', affiliate: affiliate, page: 5 }
     end
 
-    it 'should output a key based on the query, options (including affiliate id), and search engine parameters' do
-      expect(described_class.new(@valid_options).cache_key).to eq("government (site:gov OR site:mil):{:query=>\"government\", :page=>5, :affiliate_id=>#{affiliate.id}}:BingV6")
+    it 'outputs a key based on the query, options (including affiliate id), and search engine parameters' do
+      expect(described_class.new(valid_options).cache_key).to eq("government (site:gov OR site:mil):{:query=>\"government\", :page=>5, :affiliate_id=>#{affiliate.id}}:BingV6")
     end
   end
 
   describe 'instrumenting search engine calls' do
     context 'when BingV6 is the engine' do
       before do
-        @valid_options = {query: 'government', affiliate: affiliate}
-        bing_search = BingV6WebSearch.new(@valid_options)
+        valid_options = { query: 'government', affiliate: affiliate }
```

Comment:
I don't think this is right; `valid_options` should be a context `let` here, not a local variable, yes?

Reply:
Good catch - that was an oversight. Fixing...